webapp/app/forms/steps/lotse/fahrkostenpauschale.py

```diff
@@ -0,0 +1,149 @@
+
+
+from flask import render_template
+from flask_babel import lazy_gettext as _l, ngettext, _
+from wtforms.validators import InputRequired
+from wtforms import SelectField
+from flask_wtf.csrf import generate_csrf
+
+from app.model.components import FahrkostenpauschaleProps
+from app.model.components.helpers import form_fields_dict
+from app.forms import SteuerlotseBaseForm
+from app.forms.steps.step import SectionLink
+from app.forms.steps.lotse.lotse_step import LotseFormSteuerlotseStep
+from app.forms.steps.lotse.personal_data import PersonAHasDisabilityPrecondition, PersonBHasDisabilityPrecondition, ShowPersonBPrecondition, StepFamilienstand, get_number_of_users
+
+
+def calculate_fahrkostenpauschbetrag(has_pflegegrad=False, disability_degree=None, has_merkzeichen_bl=False, has_merkzeichen_tbl=False,
+                                     has_merkzeichen_h=False, has_merkzeichen_ag=False, has_merkzeichen_g=False):
+    if has_pflegegrad or has_merkzeichen_bl or has_merkzeichen_tbl or has_merkzeichen_tbl or has_merkzeichen_h or has_merkzeichen_ag:
+        return 4500
+    elif disability_degree is not None and disability_degree >= 80 or (has_merkzeichen_g and disability_degree >= 70):
+        return 900
+
+    return 0
```

Comment:
Why don't we return None but 0?

Reply:
When we render the value the text could be: "None Euro" - I know this is maybe not a possible case but just in case :grin: